web/src/stream_settings_ui.ts

```diff
@@ -1089,7 +1089,7 @@ export function toggle_view(event: string): void {
                 case "Not subscribed":
                     toggler.goto("subscribed");
                     break;
-                case "All channels":
+                case "All":
```

Comment:
How can this work correctly when the other reference to this string is translated with `$t`?

Reply:
Good catch! Added a prep commit to use `data` values which are not translated.